test/e2e/webdriver/driver.js

```diff
@@ -799,6 +801,7 @@ class Driver {
    */
   async switchToWindow(handle) {
     await this.driver.switchTo().window(handle);
+    await this.windowHandles.getCurrentWindowProperties(handle);
```

Comment:
`getCurrentWindowProperties` takes a `property` as it's first parameter

Reply:
Jeez, how did this even run properly?